kafka_service/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
```

Comment:
I think It would be better to create one single docker-compose file to simplify bootstrapping of the app. There were some problems with connection to kafka when i tried to start the application.